Guess Game.py

```diff
@@ -26,36 +23,36 @@ def again():
             break
         else:
             print ("I dont know what that is. Please type yes or no")
+                    
 def Guess():
     number = random.randint(1,100)
     answer = input("Your guess is:")
     guess=7
-    while guess<8 and guess>1:
-        while True:
-            if answer not in interval:
-                print ("Your entry is  not valid. Please type a number between 0 and 100")
-                answer = input("Your guess is:")
-            else:
-                break
-        if answer==str(number):
+    while guess<8 and guess>0:
+        if answer not in range(100):
```

Comment:
In pyhton 3 input always returns in string format and string will never be in range(100)